azurerm/resource_arm_container_group_test.go

```diff
@@ -0,0 +1,173 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMContainerGroup_basicLinux(t *testing.T) {
+	ri := acctest.RandInt()
+
+	name := fmt.Sprintf("acctestcontainergroup-%d", ri)
+	resourceGroupName := fmt.Sprintf("acctestRG-%d", ri)
+
+	config := testAccAzureRMContainerGroupBasicLinux(name, resourceGroupName, testLocation())
```

Comment:
can we pass `ri` into this method and format the name and the resource group name in there, as we do in the other test cases? I can't see any usages of the `name` and `resourceGroupName` variables in this function

Reply:
I actually did it this way for other tests (data_source_arm_public_ip_test.go). I'll change it anyway.